### PR TITLE
Update use-hybrid-modern-auth.md

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
This section is not clear at all about some of the stuff it says.  For one, it's not required/necessary to even setup an Intune App Protection policy for Outlook mobile app to work with HMA.  Second, if choosing to optionally create an Intune App Protection policy for Outlook for iOS and Android, the policy only needs to include the Outlook app, not all the ones that are stated.  These false claims with no backing are the reason I'm suggesting to just delete the bad section altogether.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ For Exchange Server mailboxes, Outlook for iOS and Android's architecture is bui
...
 
-Processing information in the Microsoft Cloud enables advanced features and capabilities, such as the categorization of email for the Focused Inbox, customized experience for travel and calendar, and improved search speed. Relying on the cloud for intensive processing and minimizing the resources required from users' devices enhances the app's performance and stability. Lastly, it allows Outlook to build features that work across all email accounts, regardless of the technological capabilities of the underlying servers (such as different versions of Exchange Server, Microsoft 365, or Office 365).
+Processing information in the Microsoft Cloud enables advanced features and capabilities, such as the categorization of email for the Focused Inbox, customized experience for travel and calendar, and improved search speed. Relying on the cloud for intensive processing and minimizing the resources required from users' devices enhances the app's performance and stability. Lastly, it allows Outlook to build features that work across all email accounts, regardless of the technological capabilities of the underlying servers (such as different versions of Exchange Server, or Office 365).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:37]:
```suggestion
Processing information in the Microsoft cloud enables advanced features and capabilities, such as the categorization of email for the Focused Inbox, customized experience for travel and calendar, and improved search speed. Relying on the cloud for intensive processing and minimizing the resources required from users' devices enhances the app's performance and stability. Lastly, it allows Outlook to build features that work across all email accounts, regardless of the technological capabilities of the underlying servers (such as different versions of Exchange Server, or Office 365).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:37]: Please change "Microsoft Cloud" to "Microsoft cloud".

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md [hunk 2]
```diff
@@ -111,30 +111,30 @@ The hybrid Modern Authentication architecture has the following technical requir
     
 3. **Exchange hybrid setup**: Requires full hybrid relationship between Exchange on-premises with Exchange Online.
 
-   - A hybrid Microsoft 365 or Office 365 organization is configured in full hybrid configuration using Exchange Classic Hybrid Topology mode and is set up as specified in the [Exchange Deployment Assistant](https://technet.microsoft.com/exdeploy).
+   - Hybrid Office 365 organization is configured in full hybrid configuration using Exchange Classic Hybrid Topology mode and is setup as specified in the [Exchange Deployment Assistant](https://technet.microsoft.com/exdeploy).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:114]:
```suggestion
   - Hybrid Office 365 organization is configured in full hybrid configuration using Exchange Classic Hybrid Topology mode and is set up as specified in the [Exchange Deployment Assistant](https://technet.microsoft.com/exdeploy).
```

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md [hunk 3]
```diff
@@ -111,30 +111,30 @@ The hybrid Modern Authentication architecture has the following technical requir
...
 
        > [!NOTE]
        > Hybrid Modern Authentication is not supported with the [Hybrid Agent](https://docs.microsoft.com/exchange/hybrid-deployment/hybrid-agent).
 
-   - Requires a Microsoft 365 or Office 365 Enterprise, Business, or Education organization.
+   - Requires an Office 365 Enterprise, Business, or Education tenant.
 
-   - The on-premises mailbox data is synchronized in the same datacenter region where that Microsoft 365 or Office 365 organization is set up. For more information about where Microsoft 365 and Office 365 data is located, visit the Microsoft Trust Center.
+   - The on-premises mailbox data is synchronized in the same datacenter region where that Office 365 organization is setup. For more about where Office 365 data is located, visit the Microsoft Trust Center.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:121]:
```suggestion
   - The on-premises mailbox data is synchronized in the same datacenter region where that Office 365 organization is set up. For more about where Office 365 data is located, visit the Microsoft Trust Center.
```

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md [hunk 4]
```diff
@@ -111,30 +111,30 @@ The hybrid Modern Authentication architecture has the following technical requir
...
+   - Outlook for iOS and Android is free for consumer usage from the iOS App store and from Google Play. However, commercial users require an Office 365 or Microsoft 365 subscription that includes the Office desktop applications: Microsoft 365 Apps for Business, Microsoft 365 Business Standard, Microsoft 365 Apps for enterprise, Office 365 Enterprise E3, Office 365 Enterprise E5, or the corresponding versions of those plans for Government or Education. Commercial users with the following subscriptions are allowed to use the Outlook mobile app on devices with integrated screens 10.1" diagonally or less: Office 365 Enterprise E1, Office 365 F1, Office 365 A1, Microsoft 365 Business Basic, and if you only have an Exchange Online license (without Office). If you only have an Exchange on-premises (Exchange Server) license, you are not licensed to use the app.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:133]:
```suggestion
   - Outlook for iOS and Android is free for consumer usage from the iOS App Store and from Google Play. However, commercial users require an Office 365 or Microsoft 365 subscription that includes the Office desktop applications: Microsoft 365 Apps for Business, Microsoft 365 Business Standard, Microsoft 365 Apps for enterprise, Office 365 Enterprise E3, Office 365 Enterprise E5, or the corresponding versions of those plans for Government or Education. Commercial users with the following subscriptions are allowed to use the Outlook mobile app on devices with integrated screens 10.1" diagonally or less: Office 365 Enterprise E1, Office 365 F1, Office 365 A1, Microsoft 365 Business Basic, and if you only have an Exchange Online license (without Office). If you only have an Exchange on-premises (Exchange Server) license, you are not licensed to use the app.
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:133]: Please capitalize the "S" in "iOS App Store".

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:37]:
```suggestion
Processing information in the Microsoft Cloud enables advanced features and capabilities, such as the categorization of email for the Focused Inbox, customized experience for travel and calendar, and improved search speed. Relying on the cloud for intensive processing and minimizing the resources required from users' devices enhances the app's performance and stability. Lastly, it allows Outlook to build features that work across all email accounts, regardless of the technological capabilities of the underlying servers (such as different versions of Exchange Server or Office 365).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:37]: Please remove the comma in the last line.

File: Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md (hunk 3 above)
Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/clients/outlook-for-ios-and-android/use-hybrid-modern-auth.md:121]: "Set up" as a verb should be two words (Microsoft Style Guide).

**Comment by mattpennathe3rd** [comment]
@JohanFreelancer9 Just an FYI, please do not remove any references to Microsoft 365 from any documentation. Office 365 has been rebranded to Microsoft 365 (though Office 365 also still exists), so all references to Microsoft 365 must remain, unless someone specifically submits a docs PR or issue pointing out that Microsoft 365 is wrong in that context.

I will fix up this PR to restore the mentions of Microsoft 365, but please keep in mind for all future work (and this applies to all products, not only Exchange).

**Comment by mattpennathe3rd** [comment]
@JohanFreelancer9 Forgive me, I misread the PR! Ignore my last message (which I deleted). Thanks for editing this. I will review.

**Comment by mattpennathe3rd** [comment]
@JeremyTBradshaw Thank you for submitting this, as we really appreciate the contribution.

I spoke with my technical contact for this area and this is not a deletion we wish to make; we provide this guidance because we want customers to utilize AAD CA and Intune to protect Outlook mobile clients when the mailboxes are located on-premises.

However, you said there are parts of the procedure that are not clear, so short of deleting this, I would very much like to hear your feedback on how we can fix and clarify it.

As an aside, your PR inadvertently rolls back a lot of recent rebranding changes we made to the content (Microsoft 365 has replaced Office 365 in many, but not all, contexts), so for that reason I'm going to close this PR rather than asking you to update it. This is not the stereotypical "We're closing this PR because we're ignoring your contribution", but is purely because I don't want some of the rebranding changes to be rolled back accidentally, as this was a large amount of effort for my team over the last 2 months.

TL;DR : I can't accept this deletion, but we want your help to clarify anything you feel is too opaque. Please submit that in a new PR (with your local copy of this repo [if you have one] freshly pulled from the public branch to merge all the rebranding changes) and someone will assist you.

P.S. This is my last day working with Microsoft, so others will be working with you on GitHub from July 1st onward. I can't thank you enough for your participation in this repo and I regret that I won't be here to work with you further. Please keep contributing, as this helps **a lot** of people.

**Comment by JeremyTBradshaw** [comment]
@mattpennathe3rd Thanks for taking the time to explain and I totally understand.  I will flag this one (the email notification I received) along with the other one where I'm still planning on installing an Exchange 2019 server to see if it still has the Exchange 2013 "learn more" links.  Eventually I'll get back to them.

Good luck with your next move(s) and thanks as well for having dealt with my grumblings so gracefully each time:).
